app/tests/src/tokenRegistryApi.js

```diff
@@ -0,0 +1,128 @@
+        signatures: [
+          {
+            signature:
+              'e0ae9a6c6cb5cfdca208751db6de351f99a5d1819c38130db18c34d558fc5d3dfcf4264a50f655cab528e7b612d1108bee2950d6448f2ccaae0f3eafddd9a408',
+            publicKey: 'd04a3088839559b64b290ce5c033cbd5f2aa23fc49aa533f9c57385e98d0ca38',
+          },
+          {
+            signature:
+              '95a7fd841ce6fbc57276e3e8d4e95a1ef9ccf0181cf55f940136dc7dac96eaaf935115b739c015725c256e32feaeb1fa0e6fa28bf680a13c76ffebf023c80f0c',
+            publicKey: 'bb0d471619d5f859eb43a973356ad6f873e87065c297208bead6aeeb134d6980',
+          },
+        ],
+      },
+      description: {
+        sequenceNumber: 0,
+        value:
+          'WMT is a utility token created to connect the unconnected and bank the unbanked. With nearly 4 billion people left offline, digital exclusion is a significant problem. Together with Input Output, World Mobile will deliver a revolutionary, scalable network in Africa and beyond. Let’s kickstart the sharing economy, reboot telecoms, and open a new world!',
+        signatures: [
+          {
+            signature:
+              'b58ad9f7407fbfc4980c7a1322853764fa1fff6eac94e8a17e22fd868faeb239119fe1a944055ad786d9bc0cd2230d4706a055cbaeeb83531b52eef1ae13c209',
+            publicKey: 'd04a3088839559b64b290ce5c033cbd5f2aa23fc49aa533f9c57385e98d0ca38',
+          },
+          {
+            signature:
+              'dab9d110d20ef1a39d88afbebd2f352b7e7d439758a3c466277850307d9a938f0abff01a4ac2838a2138d067c2f1dcf78ed4e0d957280a76e6b826e455cf800c',
+            publicKey: 'bb0d471619d5f859eb43a973356ad6f873e87065c297208bead6aeeb134d6980',
+          },
+        ],
+      },
+    }
+
+    const expected = {
+      '1d7f33bd23d85e1a25d87d86fac4f199c3197a2f7afeb662a0f34e1e776f726c646d6f62696c65746f6b656e': {
+        subject:
+          '1d7f33bd23d85e1a25d87d86fac4f199c3197a2f7afeb662a0f34e1e776f726c646d6f62696c65746f6b656e',
+        description:
+          'WMT is a utility token created to connect the unconnected and bank the unbanked. With nearly 4 billion people left offline, digital exclusion is a significant problem. Together with Input Output, World Mobile will deliver a revolutionary, scalable network in Africa and beyond. Let’s kickstart the sharing economy, reboot telecoms, and open a new world!',
+        ticker: 'WMT',
+        url: 'https://worldmobiletoken.com',
+        logoHex:
```

Comment:
this seems rather like base64 to me :thinking: but I guess that's how the original API already returns it, right?

Reply:
Refactored